### PR TITLE
added a maodou blog

### PR DESCRIPTION
**Pull request by limingth**
- [*] Add Blog Header, including title, author, date.
e.g.
```
---
layout: post
title: "如何用PadPro实现发送微信小程序"
author: limingth, zhaoic
date: '2019-07-18 09:00:00 +0800'
---

> 作者: [limingth](https://github.com/limingth)，[zhaoic](https://github.com/zhaoic) 毛豆网

**内容纲要**  
* 毛豆课堂项目背景介绍  
* 如何用PadPro实现发送微信小程序  
* 进一步需要完成的工作  
<!--more-->
```

[See more](https://github.com/Chatie/blog#contribute-guideline)

### CODE REVIEW

**Comment by huan** [comment]
Thank you very much for the great blog post article!

However, we need to fix some minor issue before we can merge it:

1. Please save all the images to the `/download/2019` directory, so that we will make sure that we can load the image from our own server in the future.
1. Please sign the CLA.

I'll be able to merge it after all the above had been done.

Have a nice weekend!

**Review by huan — CHANGES_REQUESTED** [review]
Please follow my comments.

**Comment by limingth** [comment]
@huan  those 2 requests are done!

File: _posts/2019-07-18-send-miniprogram-using-padpro.md [hunk 1]
```diff
@@ -0,0 +1,475 @@
...
+## 毛豆课堂项目背景介绍
+3个月前，也就是2019.4.18，我们公司启动了一个小程序项目 -- **毛豆课堂**。这个项目是依托[毛豆网](https://maodou.io)的音视频互动直播技术，搭建一个专注少儿素质课程的在线教育平台。
+
+目前这个小程序的MVP已经通过微信审核上线，初步实现了家长（例如孩子妈妈）给孩子的课程设定时间提醒，提醒的方式包括小程序消息，短信，电话，邮件等四种常用通知方式。转发这个课程还可以邀请和课程有关的人（例如孩子爸爸，同学家长和课程老师等）也加入这个课程提醒，未来计划还会开发一些类似家长朋友圈的社交功能。有兴趣体验的朋友，可以扫这个[毛豆课堂小程序二维码](../download/2019/maodou-ketang-invite-qrcode.png)。
+
+1个月前，我们觉得通过这个小程序创建课程提醒的流程还是不够方便，想到了能否在微信里通过聊天方式来创建提醒，例如孩子妈妈发过来一条消息`”今天晚上6点晨晨英语课，在商务会馆1101“`，收到消息的爸爸就可以转发给一个微信机器人bot，从而为爸爸建好这个课程提醒，爸爸加入后就可以提醒他去送孩子上课。有这个想法的时候，正好有一天带娃路上偶遇了李卓桓，很自然就想到了要用[Wechaty](https://github.com/Chatie/wechaty/)这个牛逼闪闪的微信机器人开源项目。
+
+其实知道这个项目已经有好几年了，但登录到github上看到[huan](https://github.com/huan)的5000多次commits还是把我惊了一跳，心中无数个草泥马翻腾而过，想说zixia还是那个zixia，你大爷居然还在写代码。Orz 钦佩之余，接下来一个多星期我也日夜奋战，虽然只有C语言和ARM汇编的基础，但在学习了[xiaoli-news-bot](https://github.com/wechaty/wechaty-getting-started/tree/master/examples/third-party/xiaoli)代码框架的基础上，实现了我们的[maodou-ketang-bot](https://github.com/maodouio/wechaty-getting-started/blob/master/examples/third-party/maodou/README.md)。有兴趣体验的朋友，可以扫这个[毛豆课堂小助手二维码](../download/2019/maodou-ketang-qrcode.png)，加好友之后，转给它那条孩子妈妈的消息。
+
+![毛豆课堂演示](../download/2019/maodou-ketang-demo.png)
```
Comment (huan) [inline _posts/2019-07-18-send-miniprogram-using-padpro.md:25]: The download-able file URLs might be changed according to different configurations.

For example, your blog post url might be:

1. `https://blog.chatie.io/send-miniprogram-using-padpro/`, or
1. `https://blog.chatie.io/blog/send-miniprogram-using-padpro/`

Like the above, two URLs demonstrated that the blog post url might at the first or the second depth of url, but the `download` directory will always at `https://blog.chatie.io/download`.

So please change your file address like the following diff shows:

Please change this line:

```diff
- [毛豆课堂演示](../download/2019/maodou-ketang-demo.png)
+ [毛豆课堂演示]({{ '/download/2019/maodou-ketang-demo.png' | relative_url }})
```

**Review by huan — CHANGES_REQUESTED** [review]
Please change the url link from hardcoded to generated by jekyll because we are going to change the site configuration today.

**Comment by huan** [comment]
I'm preparing to start working on the blog right now, so I will do the fix for you later.

**Pull request merged**